pom.xml

```diff
@@ -92,14 +87,15 @@
       <artifactId>jira-rest-java-client-api</artifactId>
       <version>${jira-rest-client.version}</version>
       <exclusions>
+        <!-- Deprecated and not needed at runtime -->
         <exclusion>
-          <groupId>com.google.guava</groupId>
-          <artifactId>guava</artifactId>
+          <groupId>com.google.code.findbugs</groupId>
+          <artifactId>jsr305</artifactId>
         </exclusion>
-        <!-- provided by commons-lang3-api -->
+        <!-- Provided by Jenkins core -->
         <exclusion>
-          <groupId>org.apache.commons</groupId>
-          <artifactId>commons-lang3</artifactId>
```

Comment:
`commons-lang3` is not a dependency of this artifact, so there is no need to exclude it.